### DISCUSSION TITLE
How to install and test louvre-views on any linux distro?

### DISCUSSION THREAD

**Discussion by renhiyama** [CuarzoSoftware/Louvre#72] [Support] [answered]
I want to test this example out, however I don't know where to start.
I also want to know if I manage to run this, do I need to run louvre compositor on my linux distro (atm fedora) ? If yes, how? I currently use gnome desktop, and I installed `louvre` package from copr as told in docs, but I dont see any desktop option in GDM desktop choosing option. Please help :)

**Comment by ehopperdietzel** [comment, marked as answer]
Hi, if you enabled [`ngompa/louvre`](https://copr.fedorainfracloud.org/coprs/ngompa/louvre/), then you can install the examples with:

`$ sudo dnf install louvre-examples`

> I dont see any desktop option in GDM

I added a `louvre.desktop` quite recently so it may not be included in the Fedora package yet, that's why the option doesn't appear in GDM.
You can enable it by creating a `/usr/share/wayland-sessions/louvre.desktop` file containing:

```
[Desktop Entry]
Name=Louvre
Comment=A macOS look-alike example compositor made with Louvre.
Exec=louvre-views
Type=Application
```

You could also directly try any example by:

- Switching to a free TTY, for example by pressing `fn + ctrl + alt + F4`.
- Log in with your user credentials.
- Launch `$ louvre-views`.
- To kill the compositor press `ctrl + shift + esc`.
- Press `fn + ctrl + alt + F2` to return to your Gnome session.
